cms/settings/base.py

```diff
@@ -929,7 +929,7 @@
 
 
 # Wagtail embeds responsive html
-WAGTAILEMBEDS_RESPONSIVE_HTML = True
+WAGTAILEMBEDS_RESPONSIVE_HTML = True  # TODO: Now that the ONSEmbedBlock is gone, I am guessing I can remove this?
```

Comment:
If this is a discussion point for review, better to leave it as a GitHub comment.

If this is to be left in the code, let's not use first person reflections and instead something like: _"TODO: Now that the ONSEmbedBlock is gone, this may be redundant"_